src/vm/dllimportcallback.cpp

```diff
@@ -1111,8 +1195,13 @@ UMEntryThunk* UMEntryThunk::CreateUMEntryThunk()
 
     UMEntryThunk * p;
 
-    // On the phone, use loader heap to save memory commit of regular executable heap
-    p = (UMEntryThunk *)(void *)SystemDomain::GetGlobalLoaderAllocator()->GetExecutableHeap()->AllocMem(S_SIZE_T(sizeof(UMEntryThunk)));
+    p = s_thunkFreeList.GetUMEntryThunk();
+
+    if (p == NULL)
+    {
+        // On the phone, use loader heap to save memory commit of regular executable heap
```

Comment:
Nit: Delete this comment. It is not relevant anymore.

Reply:
Done.